benchmarks/main.go

```diff
@@ -1,7 +1,12 @@
 package main
 
-import "github.com/estensen/runtime-systems/benchmarks/hello"
+import (
+	"github.com/estensen/runtime-systems/benchmarks/hello"
+
+	"github.com/estensen/runtime-systems/benchmarks/profiler"
+)
 
 func main() {
+	profiler.Profiler()
 	hello.Hello()
```

Comment:
If we're benchmarking a program from main, why is `hello` also run?

Reply:
I should have removed hello